src/main/java/org/umssdiplo/automationv01/core/utils/ErrorMessage.java

```diff
@@ -1,6 +1,13 @@
 package org.umssdiplo.automationv01.core.utils;
 
 public class ErrorMessage {
+    public static final String ERROR_MESSAGE_DELETE_POSITION = "%s should be deleted";
+    public static final String ERROR_MESSAGE_ASSIGN_ITEM_TITLE = "%s should be displayed";
+    public static final String ERROR_MESSAGE_FIRST_COLUMN_TEXT = "%s should be the first column";
```

Comment:
probably we can use only one error message for column text and only send the column name
public static final String ERROR_MESSAGE_FIRST_COLUMN_TEXT = "%s column should be displayed";
or something like that